test/test_plugin_helper.rb

```diff
@@ -1,7 +1,7 @@
 # This calls the main test_helper in Foreman-core
 require 'test_helper'
-
 require 'dynflow/testing'
+require 'database_cleaner'
```

Comment:
I also took the ocassion to fix this. If you're running tests individually (not with `rake test`), these tests will not work  because of DatabaseCleaner in line 11. This is only loaded on [integration tests](https://github.com/theforeman/foreman/blob/develop/test/integration_test_helper.rb#L9) now in core , so we have to require it manually.